.github/workflows/build.yml

```diff
@@ -62,6 +57,5 @@ jobs:
 
       - name: Docker build
         env:
-          MAXMIND_LICENSE_KEY: ${{ secrets.MAXMIND_LICENSE_KEY }}
           GITHUB_TOKEN: ${{ github.token }}
-        run: docker build --build-arg MAXMIND_LICENSE_KEY --build-arg GITHUB_TOKEN -t latest .
+        run: docker build --build-arg GITHUB_TOKEN --build-arg CODEBUILD_RESOLVED_SOURCE_VERSION=turn-off-max-mind -t latest .
```

Comment:
@xinaxu so I had to add [this commit](https://github.com/data-preservation-programs/ValidationBot/pull/38/commits/6d35243d43f2207fac53385546509cee6fcb22e1) in order to get this to pass as when the other ci runs, it runs against the `main` branch. 

since this passes, if I revert this commit and merge, it should run fine against main, good to move forward?